src/app/components/http-table/http-table.component.ts

```diff
@@ -0,0 +1,22 @@
+import { Component, OnInit } from '@angular/core';
+import { data } from '../../sampleData';
+
+@Component({
+  selector: 'app-http-table',
+  templateUrl: './http-table.component.html',
+  styleUrls: ['./http-table.component.scss']
+})
+export class HttpTableComponent implements OnInit {
+
+  data;
```

Comment:
This should have a type annotation. If there is currently no type for it, please create a model (either a class or an interface) for the structure you expect to receive in response.

Reply:
Made required changes.